docs/_integration-tutorial-code/example-eslint-integration.js

```diff
@@ -0,0 +1,64 @@
+/** 
+ * @fileoverview An example of how to integrate ESLint into your own tool
+ * @author Ben Perlmutter
+ */
+
+const { ESLint } = require("eslint");
+
+function createESLintInstance(eslintConfig){
+    if (eslintConfig) {
+        return new ESLint({ useEslintrc: false, overrideConfig: eslintConfig });
+    } else {
+        return new ESLint({ useEslintrc: false });
+    }
+}
+
+// Define a function that lints the specified files and returns the error results
+async function getLintingResults(eslint, filePaths) {
+  const results = await eslint.lintFiles(filePaths);
+
+  // Apply automatic fixes and output fixed code
+  await ESLint.outputFixes(results);
+
+  // Get error results
+  const errorResults = ESLint.getErrorResults(results);
```

Comment:
Is there a reason you're filtering out warnings here?

Reply:
no, will look at how to capture warnings as well, and if cannot figure out will ask for further assistance